build/alpine/Dockerfile.intermediate

```diff
@@ -12,16 +12,15 @@ ARG deps=quay.io/wire/alpine-deps
 #--- Builder stage ---
 FROM ${builder} as builder
 
-# ensure no stale files remain if they get deleted from the branch.
-RUN find /src/wire-server/ -maxdepth 1 -mindepth 1 | grep -v .stack- | xargs rm -rf
+WORKDIR /wire-server/
 
-COPY . /src/wire-server/
+COPY . /wire-server/
 
-RUN cd /src/wire-server && make clean install
+RUN make clean install
```

Comment:
do we need the `clean`?

Reply:
According to my understanding we don't; but it likely doesn't slow things down by more than a few milliseconds 🤷‍♂️

Reply:
> According to my understanding we don't; but it likely doesn't slow things down by more than a few milliseconds  I'd rather just leave it in in case we ever change the preceding scripts back.

I'm more worried about the confusion it causes to people like me who try to adjust their model of what's going on to one where it's actually necessary.  But keeping changes small is also a good thing, and it was there before, so never mind.  :)